cfgov/unprocessed/css/organisms/featured-menu-content.less

```diff
@@ -0,0 +1,110 @@
+// Duplicated from current implementation in Capital Framework v4.6.0
+// with minor edits (to be released in CFv4.7.0)
+// This can be deleted safely when this project is ported to CFv4
+
+@fcm-bp-max: 700px;
+@fcm-bp-min: 701px;
+
+.o-featured-content-module {
+  .webfont-regular();
+
+  position: relative;
+
+  min-height: 320px;
+  margin-right: -1px;
```

Comment:
Looking at it though, I'd rather do this on the `block` because that controls the borders and it looks like the left margin on the FCM is making that border disappear. I should probably address that in another PR since this is existing CF code that I'm just moving into this project.